train_without_history.sh

```
@@ -0,0 +1,24 @@
+# Section 1: Base Experiments
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)